app/controllers/account_reset/request_controller.rb

```diff
@@ -6,6 +6,8 @@ class RequestController < ApplicationController
 
     def show
       analytics.account_reset_visit
+      presenter = ConfirmationEmailPresenter.new(current_user, view_context)
+      @confirmation_period = presenter.confirmation_period
```

Comment:
Account reset should use the period from `IdentityStore.config.account_reset_wait_period_days`, not `Devise.confirm_within`?

Reply:
The way I understood this, this should use `account_reset_wait_period_days` because this is   for after the account reset is sent which could vary based on what the user sets.